site/docs/guide/deployment-types.md

```diff
@@ -136,14 +136,14 @@ ______________                                   _____________
 **The main advantage of long polling over webhooks is that it is simpler.**
 You don't need a domain or a public URL.
 You don't need to fiddle around with setting up SSL certificates in case you're running your bot on a VPS.
-Use `bot.start()` and everything will work, no further configuration required.
+Use `bot.start()`, and everything will work; no further configuration is required.
 Under load, you are in complete control of how many messages you can process.
 
 Places where long polling works well include:
 
-- During development on your local machine.
-- On majority of servers.
-- On hosted "backend" instances, i.e. machines that actively run your bot 24/7.
+- During the development of your local machine.
```

Comment:
How does this make sense? Why do you suggest that people are developing their local machine? grammY is a framework for bots, not computers. This seems increasingly like you just threw some dumb tool onto the docs without giving any thought to the wording. Maybe I'm wasting my time here?